editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2543,16 +2545,19 @@ void Node3DEditorViewport::_notification(int p_what) {
 				gpu_time += gpu_time_history[i];
 			}
 			gpu_time /= FRAME_TIME_HISTORY;
+			// Prevent division by zero for the FPS counter (and unrealistically low values).
+			// This limits the reported FPS to 10000.
+			gpu_time = MAX(0.1, gpu_time);
```

Comment:
Isn't `0.1` too high for such a lower bound? It's unlikely for a Godot game to reach beyond 10000 FPS but if someone manages to reach that on a super trivial test project, they're not going to understand why it seems stuck on 10000 and not fluctuating at all.

On my system I do reach 10000 FPS running `glxgears` for example:
```
$ vblank_mode=0 DRI_PRIME=0 glxgears
ATTENTION: default value of option vblank_mode overridden by environment.
49148 frames in 5.0 seconds = 9829.539 FPS
50753 frames in 5.0 seconds = 10150.479 FPS
```

Reply:
I'd suggest using `0.01` as a safer unrealistic lower bound. Still far enough from 0 and extremely unlikely that anyone manages to reach 100000 FPS.

Reply:
I can reach > 10,000 FPS on an empty scene with a small window. Here's what it looks like with the frame times clamped to 0.01:

![image](https://user-images.githubusercontent.com/180032/114183325-daf0ba00-9943-11eb-8539-879bf1d8545d.png)